main.tf

```diff
@@ -0,0 +1,9 @@
+resource "aws_instance" "app_server" {
+  ami           = "ami-02396cdd13e9a1257"
+  instance_type = "t2.nano"
+
+  tags = {
+    Name = "ExampleAppServerInstance"
+    Owner = "Alejandro"
+  }
+  
```

Comment:
Too many newlines at the end of the file, only one is needed

Reply:
Updated